client/js/libs/handlebars/tz.js

```diff
@@ -1,7 +1,9 @@
 "use strict";
 
 const moment = require("moment");
+const options = require("../../options");
```

Comment:
Does this happen to register options stuff twice? That code isn't so modular to be required multiple times afaik.